DataDog/chaos-controller
# Instance multiple injectors when targeting multiple containers

### What does this PR do?

It creates multiple instances of one injector when targeting multiple containers.

### Motivation

Be able to inject and clean disruptions to and from multiple containers within the same chaos pod.

### Testing Guidelines

* create a demo pod with multiple containers
* apply a disruption to it without specifying any containers

It should apply it to all of them with a single chaos pod.

### Additional Notes

The network disruption being a bit special, we need to only apply a part of it when injecting into multiple containers.
